Tasks/UsePythonVersionV0/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -5,13 +5,13 @@
   "loc.instanceNameFormat": "Use Python $(versionSpec)",
   "loc.group.displayName.advanced": "Advanced",
   "loc.input.label.versionSpec": "Version spec",
-  "loc.input.help.versionSpec": "Version range or exact version of a Python version to use.",
+  "loc.input.help.versionSpec": "Version range or exact version of a Python version to use, using semver's version range syntax. [More information](https://www.npmjs.com/package/semver)",
```

Comment:
I would have used an fwlink, but the new aka.ms tool is broken.  At any rate, when we write our own documentation I will probably rewrite this string to remove the reference to semver.